timestring/Date.py

```diff
@@ -55,9 +55,39 @@
 )
 
 
+class Specified:
+    def __init__(self, year, month, day, daytime, hour, minute, second, microsecond):
+        self.year = year
```

Comment:
'year' means more actual year value. We can assign default values to all arguments so that it is clear that these are flags not values.

Reply:
I can add type hinting like year: Boolean. I think default values would be a bit confusing.